src/styles/utils/_mixins.scss

```diff
@@ -0,0 +1,36 @@
+@mixin center-position {
+  position: relative;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+}
+
+@mixin circle($size) {
+  width: $size;
+  height: $size;
+  border-radius: 50%;
+}
+
+@mixin hand-position($hand-width) {
+  position: absolute;
+  bottom: 50%;
+  left: ($watch-size - $hand-width) / 2;
+}
+
+@mixin hand-animation($name, $duration) {
+  transform-origin: bottom center;
+  animation-name: $name;
+  animation-duration: $duration;
+  animation-timing-function: steps(60, end);
+  animation-iteration-count: infinite;
+
+  @keyframes #{$name} {
+    from {
+      transform: rotate(0deg);
+    }
+
+    to {
+      transform: rotate(360deg);
+    }
+  }
```

Comment:
you create 2 different animations, but they are the same - so do only one and set it to the both hands with the different duration 😉